framework/src/vectorpostprocessors/NodalVectorPostprocessor.C

```diff
@@ -17,8 +17,8 @@ defineLegacyParams(NodalVectorPostprocessor);
 InputParameters
 NodalVectorPostprocessor::validParams()
 {
-  InputParameters params = NodalUserObject::validParams();
-  params += VectorPostprocessor::validParams();
+  InputParameters params = VectorPostprocessor::validParams();
+  params += NodalUserObject::validParams();
```

Comment:
This is due to the changes in `InputParameters.C` in the same commit. Now if `_moose_base` is set in the parameters, when merging another set of parameters, even `_moose_base` is in there, we do not change the existing `_moose_base`. Because of this `VectorPostprocessor::validParams()` needs to be called before `NodalUserObject::validParams()`.

Reply:
What is _moose_base used for?
Is it for the registerBase call?

Reply:
`registerBase` sets `_moose_base`. It is used in `Parser` for sanity checks if I understand correctly.

Reply:
I think this is going to be confusing and a source of hard to pinpoint bugs. 
Why do you need to do this?